src/invoices/InvoiceLineDetails/OtherRelatedInvoiceLines/OtherRelatedInvoiceLines.js

```diff
@@ -81,27 +87,51 @@ const getResultFormatter = ({ search }) => ({
 
 export const OtherRelatedInvoiceLines = ({ invoiceLine, poLine }) => {
   const location = useLocation();
-  const { invoiceLines, isLoading } = useOtherRelatedInvoiceLines(invoiceLine?.id, poLine?.id);
+  const [pagination, setPagination] = useState({ limit: RESULT_COUNT_INCREMENT, offset: 0 });
+  const [
+    sortingField = DEFAULT_SORT_FIELD,
```

Comment:
`sortingField` and `sortingDirection` come with an initial value `''` (empty string) from `useSorting`. So it seems it's no use setting default value here

Reply:
Maybe it should be used in MCL prop, like sortDirection={sortDirection || DESC_DIRECTION}